### DISCUSSION TITLE
How do i enable Week 7 Video Cutscenes on Desktop

### DISCUSSION THREAD

**Discussion by GabyStarStudiosOfficial** [FunkinCrew/Funkin#2114] [Q&A] [answered]
I Started to Compile the FNF WEEK 7 Source Code When i started Story Mode The Cutscene didn't begin, It was Pitch Black. But the Healthbar and icons are still there. Is there a way to Fix it?

**Comment by AbnormalPoof** [comment, marked as answer]
FlxVideo (the shit the week 7 cutscenes use) only works on HTML5, you would need to use something like hxCodec to get videos on desktop.

**Reply by FunkoDev** [reply]
WOAHH

**Comment by GabyStarStudiosOfficial** [comment]
I'm There and i got HxCodec Now what am i suppose to do now?
